nservicebus/operations/metrics-service-control.md

```diff
@@ -0,0 +1,17 @@
+---
+title: Metrics data to ServiceControl
+summary: Reporting metrics to ServiceControl.
+reviewed: 2017-09-26
+component: MetricsServiceControl
+related:
+ - nservicebus/operations
+related:
+ - samples/logging/metrics
+---
+
+## WIP 
```

Comment:
Could you write the content of this page? I know that we wait for the release decision, but once the content is done and a package released we can move forward with it.

Reply:
I had a session with @tmasternak (Thanks Tomasz!) and pushed an updated version now